src/lib/rsa/sandboxed/RSAKeysIframe.html

```diff
@@ -3,7 +3,8 @@
 <head>
     <meta charset="UTF-8">
     <!-- Note: sandbox and frame-ancestors policies can not actually be specified via a meta tag. You have to send them
-    as response headers. -->
+    as response headers. Additionally, the server needs to include the sandboxed iframe's null origin in its cors header
+    for the script files imported here, in order to pass integrity checks added by the build process. -->
```

Comment:
What does that mean? What must the server send now? Can you list that here so we can tell IT?